frontend/server/controllers/ContestController.php

```diff
@@ -230,6 +228,7 @@ public static function getContestListInternal(Request $r, $callback_user_functio
      * @throws InvalidDatabaseOperationException
      */
     public static function apiMyList(Request $r) {
+        self::authenticateRequest($r, true /* validateRealUser */);
```

Comment:
```suggestion
        self::authenticateRequest($r, true /* requireMainUserIdentity */);
```

igual en el resto del cambio.

Reply:
Listo